lib/Translator.js

```diff
@@ -0,0 +1,248 @@
+'use strict';
+
+const worldStateData = require('warframe-worldstate-data');
+
+/**
+ * Translator to convert in-game names to their localizations
+ */
+class Translator {
+		
+	constructor(lang = 'en_US') {
+		this.data = worldStateData[lang];
+		if(typeof this.data !== "object") {
```

Comment:
we wouldn't want to completely make the translator unavailable because of an incorrect iso language selection, that's why i'm saying defaulting back to the default (en_US) if it doesn't exist. it's a redundancy against mistaking a symbol or something similar.